Resources/Private/Fusion/Layouts/AmpPage.fusion

```diff
@@ -1,4 +1,16 @@
 prototype(Shel.Blog:Layout.AmpPage) < prototype(Neos.Neos:Page) {
+    # Disable Neos.GoogleAnalytics and include amp tag manager -> https://support.google.com/tagmanager/answer/9205783?hl=en
+    googleTagManagerNoScript >
+    googleTracking >
+    googleTracking = Neos.Fusion:Join {
+        main = afx`
+            <script async custom-element="amp-analytics" src="https://cdn.ampproject.org/v0/amp-analytics-0.1.js"></script>
+            <amp-analytics config={'https://www.googletagmanager.com/amp.json?id=' + Configuration.setting('Shel.Blog.analytics.tagManager.id') + '&gtm.url=SOURCE_URL'} data-credentials="include"></amp-analytics>
+        `
+        @if.isset = ${Configuration.setting('Shel.Blog.amp.analytics.tagManager.id')}
```

Comment:
Wouldn't it make sense to take the setting from the Analytics package setting path?
Then it's not necessary to define it twice.

And did you use a `Join` because people can add more to it?

Reply:
Inside google tag manager you have to create your own container type for amp pages, which then has its own container ID. Perhaps the settings name could be better, something like ampContainerID.

I've used a `Join`  to get access to the `@position` method. 
if you have a better idea i would be happy if you give me some idea how i can do it better

Reply:
You can always use the `@position`. I think in this case I wouldn't even use AFX but just a `Neos.Fusion:Tag` instead of the join and define the attributes.